pkgs/by-name/do/dooit-extras/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  python311,
+  dooit,
+  nix-update-script,
+}:
+let
+  python3 = python311;
+  version = "0.2.0";
+in
+python3.pkgs.buildPythonPackage {
+  pname = "dooit-extras";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dooit-org";
+    repo = "dooit-extras";
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
This is a new way of handling tags. What do you mean it doesn't work without rev?

Reply:
Got this error while building:

```
       error: function 'anonymous lambda' called without required argument 'rev'
       at /nix/store/frfyxcpzsdasdin76x83krbhpgkis8b0-source/pkgs/build-support/fetchgithub/default.nix:4:1:
            3| lib.makeOverridable (
            4| { owner, repo, rev, name ? "source"
             | ^
            5| , fetchSubmodules ? false, leaveDotGit ? null
```